src/services/productService.js

```diff
@@ -0,0 +1,27 @@
+const productDao = require('../models/productDao');
+
+const getProducts = async(productId,userInfo,productInfo)=>{
```

Comment:
```suggestion
const getProducts = async(productId, userInfo) => {
```
productInfo라는 변수는 사용되지 않고 있는 것 같아서, 제거하셔도 될 것 같습니다